x-pack/legacy/plugins/siem/public/containers/case/use_delete_cases.tsx

```diff
@@ -59,7 +59,7 @@ const dataFetchReducer = (state: DeleteState, action: Action): DeleteState => {
   }
 };
 
-interface UseDeleteCase extends DeleteState {
+export interface UseDeleteCase extends DeleteState {
```

Comment:
nit: On line 120, somehow the argument `caseIds` appears as `any`, would be clearer if we add a type for  it `caseIds: DeleteCase[]`